README.md

```diff
@@ -0,0 +1,47 @@
+# Solution_01 Answers to Assignment Part 1
+
+a) The command 'git status' does output the status of the working directory and the staging area (eg which files are changed).
+
+b) In SwiftUI, anything that gets drawn to a screen, is a view.
+
+c) print("Hello world") is an example of a global function call. viewModel.getData() is an example of an instance method call.
+
+d) Some Views: Text, Button, Slider, Image
+
+e) To create a new local repository: git init project-directory
+
+f) To Preview an app in multiple orentations: ContentView().previewInterfaceOrientation(.portrait) ContentView().previewInterfaceOrientation(.landscapeLeft) ...
+
+g) An App ist made up of a combination of classes and structs that contain properties and methods.
+
```

Comment:
Looking here for "of instances" "that contain data and functionality". However, this question seems to be worded poorly. I will take it back to the team.